src/main/scala/cromwell/util/TryUtil.scala

```diff
@@ -6,11 +6,12 @@ import scala.util.{Failure, Try}
 
 object TryUtil {
   def stringifyFailures[T](failure: Try[T]): String = {
-    val writer = new PrintWriter(new StringWriter())
+    val stringWriter = new StringWriter()
```

Comment:
If the guy who wrote the scala cookbook doesn't seem to think the flush/close is necessary, it probably isn't: http://alvinalexander.com/scala/how-convert-stack-trace-exception-string-print-logger-logging-log4j-slf4j